token/jwt/token.go

```diff
@@ -37,6 +37,12 @@ const (
 	JWTHeaderTypeValue = "JWT"
 )
 
+var SupportedSignatureAlgorithms = []jose.SignatureAlgorithm{
+	SigningMethodNone,
```

Comment:
Could we have missed some here that would cause some breaking changes in existing systems because the validation has changed?

Reply:
I listed everything standardized and available and secure. In the worst case though, this is a publicly accessible variable in Go, so people can extend it if we did miss something. Not nice API though.

But as I mentioned in my PR description, I really think we should do a backwards incompatible change after this, to address current code rot around JWTs in fosite and address some vulnerabilities available exactly because we allow for example to try all algorithms and not just the one expected. But this is probably something only you can decide on how much breaking change would you approve (not in terms of external users changes, but in terms of API changes for fosite integrators - so some fosite structs we could just get rid of and delegate to go-jose).

Reply:
I'm pretty sure we validate the signing method somewhere, do we not?